tests/ci_install.sh

```diff
@@ -30,43 +30,34 @@ cp tests/matplotlibrc .
 
 # Step 1: pre-install required packages
 if [[ "${RUNNER_OS}" == "Windows" ]] && [[ ${dependencies} != "minimal" ]]; then
-    # Install some dependencies using conda (if not doing a minimal run)
-    CYTHON=$(grep cython tests/test_prerequirements.txt)
-    NUMPY=$(grep numpy tests/test_prerequirements.txt)
-
-    CARTOPY=$(grep cartopy tests/test_requirements.txt)
-    H5PY=$(grep h5py tests/test_requirements.txt)
-    MATPLOTLIB=$(grep matplotlib tests/test_requirements.txt)
-    SCIPY=$(grep scipy tests/test_requirements.txt)
-    conda config --set always_yes yes
-    conda info -a
-    conda install --quiet --yes -c conda-forge \
-      $CYTHON $NUMPY $CARTOPY $H5PY $MATPLOTLIB $SCIPY
+    # windows_conda_requirements.txt is a survivance of test_requirements.txt
+    # keep in sync: setup.cfg
+    while read requirement; do conda install --yes $requirement; done < tests/windows_conda_requirements.txt
```

Comment:
```suggestion
    conda install --yes --file tests/windows_conda_requirements.txt
```

Reply:
I don't think it works unfortunately. To the best of my knowledge, Conda can only install an named environment from a yaml file, and I didn't want to invest any more effort in a temporary band aid